source/analysis/dynamic_analysis.py

```diff
@@ -502,7 +505,7 @@ def animate_skin_model_time_history(self, skin_model_params):
             for idx, label in zip(list(range(GD.DOFS_PER_NODE[self.structure_model.domain_size])),
                                   GD.DOF_LABELS[self.structure_model.domain_size]):
                 start = idx
-                step = GD.DOFS_PER_NODE[self.structure_model.domain_size]
```

Comment:
global definintions should be importad as GD and not wildcard import to know where the values come from.